tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceUITest.java

```diff
@@ -189,45 +189,31 @@ public void testChangeInputForIncrementalSearch() {
 	@Test
 	public void testFindWithWholeWordEnabledWithMultipleWords() {
 		initializeTextViewerWithFindReplaceUI("two words two");
-		dialog.setFindText("two");
+		dialog.select(SearchOptions.INCREMENTAL);
```

Comment:
Is this change necessary? If yes, it may be worth to be provided in a separate PR with an according explanation.

Reply:
I want to keep the changes in the Overlay-PR to a minimum. Since the Overlay does NOT have an incremental-option (all searches are incremental unless that is not possible), I need to add this option so both Interfaces perform the same operations.

As discussed, I will add a Test which is exclusive to the Dialog, which tests the same behavior but without Incremental